com.unity.render-pipelines.universal/Runtime/Passes/ScriptableRenderPass.cs

```diff
@@ -229,6 +244,24 @@ public void ConfigureInput(ScriptableRenderPassInput passInput)
             m_Input = passInput;
         }
 
+        public void ConfigureColorStoreAction(uint attachmentIndex, RenderBufferStoreAction storeAction)
```

Comment:
i'd suggest to change this to ConfigureColorStoreAction(RenderBufferStoreAction storeAction, uint attachmentIndex = 0), as most of the cases are going to be single target

Reply:
we discussed to keep this interface as I think more explicit is less error prone